diff --git a/config/routes.rb b/config/routes.rb
index 26fb2fc..02cf969 100644
--- a/config/routes.rb
+++ b/config/routes.rb
@@ -49,7 +49,7 @@ Rails.application.routes.draw do
     resources :notes, except: [:new]
     resource :cache, controller: 'cache', only: [:show, :destroy]
     resources :campaigns, except: [:show]
-    
+
     resources :contents, path: :articles do
       collection do
         get :auto_complete_for_article_keywords
@@ -89,7 +89,7 @@ Rails.application.routes.draw do
 
     # Looks like removal has been omitted from the interface
     resources :users, except: :destroy
-    
+
     get '/settings/general', to: 'settings#general', as: :general_settings
     get '/settings/write', to: 'settings#write', as: :write_settings
     get '/settings/feedback', to: 'settings#feedback', as: :feedback_settings
@@ -110,5 +110,6 @@ Rails.application.routes.draw do
 
   root to: 'articles#index', format: false
 
+  get '*from/amp', to: 'amp_articles#show', as: :amp_article
   get '*from', to: 'articles#show', as: :article
 end
